src/main/java/pages/ResizablePage.java

```diff
@@ -14,6 +14,11 @@
 public class ResizablePage extends BasePage {
 
     private final static Logger LOG = Logger.getLogger(ResizablePage.class);
+    private final static int OFFSET = 50;
```

Comment:
It's better to keep such values in constant. In this case we will never be messed with values.